exercises/acronym/example.rs

```diff
@@ -0,0 +1,20 @@
+pub fn abbreviate(phrase: &str) -> String {
+    phrase.split(|c: char| c.is_whitespace() || !c.is_alphanumeric())
+          .flat_map(|word| split_camel(word))
+          .filter(|word| !word.is_empty())
```

Comment:
Interesting that `char.to_uppercase()` does that. I assume that this line replaces the `filter` and `map` lines, but not the `collect` line. So now there are two `collect`! What if we tried to avoid that by taking advantage of `String#to_uppercase`(https://doc.rust-lang.org/std/string/struct.String.html#method.to_uppercase) ? (You probably have to put the type hint back, that's fine).

Ultimately I'm OK with that `filter_map` as-is but let's see if we can `collect` fewer times, eh?